signing_service/src/signing_service.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import logging.config
+import os
+import socket
+
+
+logger = logging.getLogger()
+crash_logger = logging.getLogger('crash')
+
+
+def main(concent_cluster_address):
+    address, port = concent_cluster_address.split(':')
+    port = int(port)
+
+    socket_to_concent = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    socket_to_concent.connect((address, port))
+
+    socket_to_concent.send(b'GET / HTTP/1.0\r\n\r\n')
+    data = socket_to_concent.recv(1024)
+    logger.info(data)
+    socket_to_concent.close()
+
+
+def _parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('concent_cluster_address', help='Address of a Concent cluster')
+
+    return parser.parse_args()
```

Comment:
I think good thing might be validation to address which is giving by this function

Reply:
I will add it in https://github.com/golemfactory/concent/pull/634, this is just a prove of concept. Btw. how would you like this validation to be done ? Validate if is string ? Or if it is a host or ip ?

Reply:
Validation would be a good idea. But if you don't have validation please at least catch and handle the exception that is raised by the socket when you give it an invalid address

This issue was meant to be simple so you don't have to do it here. In #634 is fine.

Reply:
Yes, I prefer to do it in #634

Reply:
:+1: done